dsl/scripts/jobs.groovy

```diff
@@ -143,6 +143,90 @@ def jobs = [
             stringParam('DEVOPSSCRIPTS_BRANCH', 'main', 'Git branch, tag or hash of the vegaprotocol/devopsscripts repository')
             stringParam('ANSIBLE_BRANCH', 'master', 'Git branch, tag or hash of the vegaprotocol/ansible repository')
         }
+    ],
+    [
+        name: 'private/Deployments/Veganet/Stagnet',
+        useScmDefinition: false,
+        definition: {
+            cps {
+                script("""
+                @Library('vega-shared-library@main') _
+                pipelineDeploy()
+                """)
+            }
+        },
+        env: [
+            NET_NAME: 'stagnet',
+        ],
+        parameters: {
+            stringParam('VEGA_CORE_VERSION', 'develop', "Git branch, tag or hash of the vegaprotocol/vega repository. Leave empty to not deploy a new version of vega core. If you decide not to build binary by yourself you need to set version according to the versions available on releases page: https://github.com/vegaprotocol/vega/releases")
```

Comment:
I think we need to keep it empty in order to not compile the vega core binary, right?